strawberry/extensions/runner.py

```diff
@@ -20,10 +20,25 @@ class ExtensionsRunner:
     def __init__(
         self,
         execution_context: ExecutionContext,
-        extensions: List[Extension] = None,
+        extensions: List[Union[Type[Extension], Extension]] = None,
     ):
         self.execution_context = execution_context
-        self.extensions = extensions or []
+
+        if not extensions:
+            extensions = []
+
+        init_extensions: List[Extension] = []
+
+        for extension in extensions:
+            if inspect.isclass(extension):
```

Comment:
Why not use `isinstance(extension, type)`?

Reply:
Yep thats better